spec/models/group_spec.rb

```diff
@@ -1,5 +1,38 @@
 require 'rails_helper'
 
 RSpec.describe Group, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:group) { Group.new }
+  let(:user) { User.new(name: 'Base User', email: 'teste@teste.com') }
+
+  context 'When creating a invalid group' do
+    it 'Name cannot be nil or empty' do
+      group.name = ''
+      group.owner_id = 1
+      expect(group).to be_invalid
+    end
+    it 'Owner_id cannot be nil or empty' do
+      group.owner_id = nil
+      group.name = 'Test Growp'
+      expect(group).to be_invalid
```

Comment:
We are going to have few discussions about testing but here it'd be better to check what is wrong with this object to avoid false positive results.

My suggestion is something like:

```suggestion
      expect(group).to be_invalid
      expect(group.errors.full_messages).to contain_exactly('Owner must exist')
```

This way we guarantee that the object is invalid but also why it is invalid.

If you apply this change before[ my previous comment](https://github.com/wfbraga/finco/pull/2/files#r885595026) you can see that you would catch that error duplication beforehand:

```ruby
  1) Group When creating a invalid group Owner_id cannot be nil or empty
     Failure/Error: expect(group.errors.full_messages).to contain_exactly('Owner must exist')
     
       expected collection contained:  ["Owner must exist"]
       actual collection contained:    ["Owner can't be blank", "Owner must exist"]
       the extra elements were:        ["Owner can't be blank"]
```